lib/smart_answer_flows/energy-grants-calculator/outcomes/_warm_home_discount.govspeak.erb

```diff
@@ -1 +1 @@
-[Warm Home Discount scheme](/the-warm-home-discount-scheme) - £<%= calculator.warm_home_discount_amount %> discount on your energy bills
+- [Warm Home Discount scheme](/the-warm-home-discount-scheme) - £140 discount on your energy bills
```

Comment:
@selfthinker Is that where that came from? Sorry. I didn't realise. #2711 Is probably going to be overwritten by all of the energy-grants-calculator changes as they come back from factcheck.

It's going to be easier to wait until they're all done, and then replay the changes.